client/react-client-app/e2e-base.Dockerfile

```
@@ -0,0 +1,23 @@
+# E2E Base Dockerfile
+# Contains ONLY heavy dependencies: Playwright browsers, ffmpeg, openssl
+# This image is cached for 30 days to avoid costly rebuilds
+# App dependencies and source code are mounted at runtime
+
+FROM mcr.microsoft.com/playwright:v1.52.0-noble
+
+WORKDIR /app
+
+# Install system tools (ffmpeg for video, openssl for certs if needed)
+RUN apt-get update && \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)